framework/include/outputs/OutputWarehouse.h

```diff
@@ -189,6 +189,43 @@ class OutputWarehouse
    */
   void bufferConsoleOutputsBeforeConstruction(bool buffer) { _buffer_action_console_outputs = buffer; }
 
+public:
+  /**
+   * Calls the initialSetup function for each of the output objects
+   * @see FEProblemBase::initialSetup()
+   */
+  void initialSetup();
```

Comment:
It would be nice if we could claim these setups as public, otherwise a class has to be claimed as friend of OutputWarehouse if the class want to access these routines.  Originally, I want to implement a SlepcOutput which need to access these setups, but later on I have to discard the implementation for some reason. 

I will change these routines back to private. In the future, we will claim them as public again when necessary.